frontend/src/component/changeRequest/ChangeRequestSidebar/EnvironmentChangeRequest/EnvironmentChangeRequest.tsx

```diff
@@ -56,146 +61,165 @@ const ChangeRequestContent = styled(Box)(({ theme }) => ({
 export const EnvironmentChangeRequest: FC<{
     environmentChangeRequest: ChangeRequestType;
     onClose: () => void;
-    onReview: (id: number, comment?: string) => void;
+    onReview: (
+        id: number,
+        willOverwriteStrategyConfig: boolean,
```

Comment:
I also think we should be able to remove this parameter and instead access the data via plausible as mentioned in a previous comment. But again, that assumes that we can make the useContext call work within the useChangeRequestApi code.